stdlib/3/concurrent/futures/process.pyi

```diff
@@ -1,46 +1,18 @@
 # Stubs for concurrent.futures.process (Python 3.5)
 #
-# NOTE: This dynamically typed stub was automatically generated by stubgen.
 
-from typing import Any
-from . import _base
+from typing import Any, Callable, TypeVar, Iterable
+from ._base import Future, Executor
 
 EXTRA_QUEUED_CALLS = ... # type: Any
 
-class _RemoteTraceback(Exception):
-    tb = ... # type: Any
-    def __init__(self, tb): ...
-
-class _ExceptionWithTraceback:
-    exc = ... # type: Any
-    tb = ... # type: Any
-    def __init__(self, exc, tb): ...
-    def __reduce__(self): ...
-
-class _WorkItem:
-    future = ... # type: Any
-    fn = ... # type: Any
-    args = ... # type: Any
-    kwargs = ... # type: Any
-    def __init__(self, future, fn, args, kwargs): ...
-
-class _ResultItem:
-    work_id = ... # type: Any
-    exception = ... # type: Any
-    result = ... # type: Any
-    def __init__(self, work_id, exception=None, result=None): ...
+class BrokenProcessPool(RuntimeError): ...
 
-class _CallItem:
-    work_id = ... # type: Any
-    fn = ... # type: Any
-    args = ... # type: Any
-    kwargs = ... # type: Any
-    def __init__(self, work_id, fn, args, kwargs): ...
+_T = TypeVar('_T')
 
-class BrokenProcessPool(RuntimeError): ...
+class ProcessPoolExecutor(Executor):
+    def __init__(self, max_workers: int = ...) -> None: ...
```

Comment:
Actually a default `= None` has a special meaning according to PEP 484 (and mypy implements this too): it modifies the type to become `Optional[]`. Since this is documented as `max_workers=None` you should either keep the default `= None` or change the type to `Optional[int]`.